index.js

```diff
@@ -86,8 +88,55 @@ module.exports = postcss.plugin('postcss-bem-linter', function(primaryOptions, s
       });
     }
 
+    function checkGlob(file, globs) {
+      // PostCSS turns relative paths into absolute paths
+      file = path.relative(process.cwd(), file);
+      return minimatchList(file, globs);
+    }
+
+    function isImplicitComponent(file) {
+      if (Array.isArray(config.implicitComponents)) {
```

Comment:
Using `Array.isArray` would require us to not support lower than Node 4. I'm happy to do that, but it would mean a major version bump in semver — and if we're doing a major version bump, there are some other things that are probably worth including (e.g. pruning the options so there aren't multiple ways of doing things). So you can you use features that are compatible with Node 0.12?